components/brave_wallet/browser/brave_wallet_utils.h

```diff
@@ -193,6 +192,11 @@ mojom::OriginInfoPtr MakeOriginInfo(const url::Origin& origin);
 
 bool IsFilecoinKeyringId(const std::string& keyring_id);
 
+bool IsBitcoinKeyring(const std::string& keyring_id);
+bool IsBitcoinNetwork(const std::string& network_id);
+bool IsValidBitcoinNetworkKeyringPair(const std::string& network_id,
```

Comment:
unit tests for these three?

Reply:
https://github.com/brave/brave-core/pull/17978/commits/2f117c534c8236292c2bbc209de5195830ccb9c5